central_v2/src/App.tsx

```diff
@@ -9,26 +9,16 @@ import { useAPIClients, Environment, AppType } from '@righton/networking';
 import { GoogleOAuthProvider } from '@react-oauth/google';
 import { ThemeProvider, StyledEngineProvider } from '@mui/material/styles';
 import { APIClientsContext } from './lib/context/APIClientsContext';
-import UserProfileReducer from './lib/reducer/UserProfileReducer';
 import Theme from './lib/Theme';
 import AppSwitch from './switches/AppSwitch';
 import CreateQuestionLoader from './loaders/CreateQuestionLoader';
```

Comment:
I'm removing UserProfileContext and replacing it with CentralDataContext, which will incoporate UserProfile as well as all other state variables that lean on an API request